src/main/java/blackjack/BlackJackApplication.java

```diff
@@ -1,21 +1,26 @@
 package blackjack;
 
-import blackjack.domain.BlackJackGame;
+import blackjack.domain.game.BlackJackGame;
 import blackjack.domain.card.Card;
-import blackjack.domain.card.CardFactory;
+import blackjack.domain.card.Deck;
+import blackjack.dto.GameResultDto;
 import blackjack.view.InputView;
 import blackjack.view.OutputView;
 
 public class BlackJackApplication {
```

Comment:
메인 클래스에 변경이 발생하지 않도록 만들어봅시다.

```suggestion
public class BlackJackApplication {
    public static void main(String[] args) {
        BlackJackGame blackJackGame = new BlackJackGame();
        blackJackGame.play();
    }
}
```

Reply:
지금 main이 입력~출력 흐름을 담당하고 있다는 관점에서 컨트롤러로 사용되고 있는데 뷰 관련 로직을 BlackJackController를 따로 만들어서 처리해야 한다는 말일까요?